src/client/routes/PacketDetailsPage/index.js

```diff
@@ -64,12 +70,26 @@ const PacketDetailsPage = () => {
     if (signURL) window.location.assign(signURL)
   }
 
+  const handleOpenSignFrame = async () => {
+    const signURL = await generateSignURL() + '&withinIframe=true'
+    setSignURL(signURL)
+    setIsSignFrameOpen(!isSignFrameOpen)
+  }
+
+  const handleOpenSignModal = async () => {
+    const signURL = await generateSignURL() + '&withinIframe=true'
+    setSignURL(signURL)
+    setIsModalOpen(true)
+  }
+
   const handlePacketDownload = () => {
     const { documentGroup } = packetDetails
     const { eid: documentGroupEid } = documentGroup || {}
     window.location.assign(`/api/packet/download/${documentGroupEid}`)
   }
 
+  const handleSignFinish = (url) => window.location.assign(url)
```

Comment:
On finish, it'd be nice to log the url it returned, close the modal or hide the frame, then refresh the packet data. I think only the button that redirects to the sign url should do any redirecting...

Reply:
Sounds good. Updated the `handleIframeSignFinish` and `handleModalSignFinish` to log the redirectURL, hide the frame/modal, then refetch packetData.

But since we're not triggering any redirect, no redirect query params would be received.

Reply:
Ah, can you parse them and pipe them into whatever shows them?

Reply:
Done